global.php

```diff
@@ -44,7 +44,7 @@ function StripMagicQuotes(&$var) {
 }
 	foreach ($EGPCS as $k => $v)
 		StripMagicQuotes($$v);
-} 
```

Comment:
Same with this commit, it's not yet in master.

Reply:
Ditto